### ISSUE TITLE
AD diffusion: inference bypasses training-time instance normalization

### ISSUE THREAD

**Issue by PansulBhatt** [NVIDIA/NV-Tesseract#40]
## Area

`ad_diffusion`

## Environment

- OS: Linux 5.15.0-1110-azure
- Python: 3.12.13
- Revision inspected: `da2c4d0`
- Installation: source checkout with the `ad_diffusion` package on `PYTHONPATH`

## What happened

The diffusion denoiser receives different input scales during training and inference.

`TSDiffuser_base.calc_loss()` applies per-window instance normalization before constructing the noisy and conditioned denoiser inputs:

```python
data_mean = observed_data.mean(dim=(1, 2), keepdim=True)
data_std = observed_data.std(dim=(1, 2), keepdim=True) + 1e-5
observed_data = (observed_data - data_mean) / data_std
observed_data = torch.clamp(observed_data, min=-10.0, max=10.0)
```

However, both `TSDiffuser_base.evaluate()` and `TSDiffuser_base.evaluate_with_dpm()` pass the raw SDK-preprocessed `observed_data` directly to `impute()` / `dpm_solver_impute()`.

For the same deterministic window:

```python
observed_data = torch.tensor(
    [[[0.0, 5.0, 10.0, 15.0], [20.0, 25.0, 30.0, 35.0]]]
)
```

the training path presents standardized values to the denoiser, while the inference path presents the original `0..35` values. The mismatch occurs before thresholding or anomaly-score post-processing and affects both standard diffusion inference and DPM-Solver inference.

## Expected behavior

Training and inference should use the same input-scale contract at the denoiser boundary. Returned reconstructions should remain in the SDK target scale so residual computation continues comparing like-for-like values.

## Impact

The published inference paths can evaluate the denoiser on a distribution different from the one used by `calc_loss()`. This affects reconstruction values and therefore the residual anomaly scores consumed by downstream threshold strategies.
